core/commands/root.go

```diff
@@ -25,8 +25,8 @@ var Root = &cmds.Command{
 BASIC COMMANDS
   init          Initialize ipfs local configuration
   add <path>    Add a file to ipfs
-  cat <ref>     Show ipfs object data
-  get <ref>     Download ipfs objects
+  cat <ref>     Show IPFS object data
```

Comment:
why capitalized here where elsewhere youre making them lowercase in the taglines?